server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_5__add_dlm_job.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE IF NOT EXISTS `dlm_job` (
+  `id` bigint AUTO_INCREMENT NOT NULL COMMENT 'ID of the dlm job object',
+  `schedule_task_id` bigint NOT NULL COMMENT 'ID of the related schedule task, refer to schedule_task.id',
+  `dlm_job_id` varchar(120) NOT NULL COMMENT 'Unique identifier of the dlm job object',
+  `table_name` varchar(120) NOT NULL COMMENT 'Name of the source table',
+  `fire_time` varchar(120) NOT NULL COMMENT 'The actual time the trigger fired',
+  `target_table_name` varchar(120) NULL COMMENT 'Name of the target table',
+  `source_datasource_info` bigint NOT NULL COMMENT 'Info of the source datasource',
```

Comment:
it is hard understand `source_datasource_info` with bigint

Reply:
corrected to varchar